docs/recipes/install/common/enable_openchami_repo.tex

```diff
@@ -0,0 +1,16 @@
+To begin we need to add the OpenCHAMI RPM to the SMS node, by adding it to the list of
+availbe packages locally. This requires network access from the {\em master}
+node to the internet.
+
+% begin_ohpc_run
+% ohpc_validation_newline
+% ohpc_comment_header Enable OpenCHAMI repositories \ref{sec:enable_confluent}
+\begin{lstlisting}[language=bash,keywords={},basicstyle=\fontencoding{T1}\fontsize{8.0}{10}\ttfamily,
+	literate={VER}{\OHPCVerTree{}}1 {OSREPO}{\OSTree{}}1 {TAG}{\OSTag{}}1 {ARCH}{\arch{}}1 {-}{-}1]i
+[sms](*\#*) PI_URL="https://api.github.com/repos/openchami/release/releases/latest"release_json=$(curl -s "$API_URL")
+[sms](*\#*) key_url=$(echo "$release_json" | jq -r '.assets[] | select(.name | endswith(".asc")) | .browser_download_url' | head -n 1)
+[sms](*\#*) key_name=$(echo "$release_json" | jq -r '.assets[] | select(.name | endswith(".asc")) | .name' | head -n 1)
+[sms](*\#*) curl -L -o "$key_name" "$key_url"
+[sms](*\#*) rpm --import "$key_name"
```

Comment:
Is there a simpler way to include the key. The confluent recipe is a good example how this could look like. Having a release RPM somewhere with a key included would be easier than parsing GitHub json files. At least from a end-user perspective.